server/src/services/socrataClient.py

```diff
@@ -0,0 +1,51 @@
+"""
+This is a simple wrapper for Socrata that handles a couple of issues:
+1. retrying requests in the event of a timeout or other failure
+2. grabbing the dataset id from the config based on the year
+3. automatically closing the client when we're done using it
+
+Usage is like this:
+    socrata = SocrataClient()
+    results = socrata.get(year, **kwargs)
+
+kwargs are all of the normal socrata kwargs - select, limit, etc.
+"""
+
+from sodapy import Socrata
+from configparser import ConfigParser
+
+
+class SocrataClient:
+    def __init__(self):
+        config = ConfigParser()
```

Comment:
This class should be able to be initialized with settings passed in from the app.py level
The config would be passed into socrata client here
https://github.com/hackforla/311-data/pull/479/files#diff-c5ff603097380b44c2051acbc55f02ccR27

Reply:
done